src/randoop/operation/ConstructorCall.java

```diff
@@ -170,11 +161,12 @@ public boolean equals(Object o) {
     if (!(o instanceof ConstructorCall))
```

Comment:
See comment elsewhere about equals methods.  This instanceof test should be first, and then the null test is not needed any more.  (The `this == o` test is still worthwhile; it could go before or after the instanceof test.)